### PR TITLE
fix compile issue with new camel version

### PR DESCRIPTION
**Pull request by jstrachan**

### CODE REVIEW

**Comment by jstrachan** [comment]
[merge]

**Comment by jstrachan** [comment]
ok to test

**Comment by jstrachan** [comment]
[merge]

File: pom.xml [hunk 1]
```diff
@@ -76,7 +76,7 @@
     <fabric8.mode>kubernetes</fabric8.mode>
 
 
-    <fabric8.version>2.2.168</fabric8.version>
+    <fabric8.version>2.2.2.2.169</fabric8.version>
```
Comment (davsclaus) [inline pom.xml]: Its 2.2.169 ;)

**Comment by jstrachan** [comment]
ok to test

**Comment by jstrachan** [comment]
[merge]

**Pull request merged**